### PR TITLE
Added RBAC documentation

### PR DESCRIPTION
**Pull request by trmartin4**
## Objective

Added documentation for Bitwarden Portal RBAC.  I also changed the name from Admin Portal to Bitwarden Portal to reflect current terminology.

### CODE REVIEW

File: docs/getting-started/server/admin.md [hunk 1]
```diff
@@ -1,19 +1,51 @@
...
 
-## Configuring users
+## Configuring access
 
-Admin Portal authentication is done entirely through a passwordless flow, using a link sent through
-email. The email address must be listed in the `adminSettings:admins` user secret to be authorized.
+### Authentication
+
+Bitwarden Portal authentication is done entirely through a passwordless flow, using a link sent
+through email. The email address must be listed in the `adminSettings:admins` user secret to be
```
Comment (Hinton) [inline docs/getting-started/server/admin.md:8]: This needs to be updated.

File: docs/getting-started/server/admin.md [hunk 2]
```diff
@@ -1,19 +1,51 @@
...
+authorized.
 
 If you’ve followed the [Server Setup Guide](./guide.md) this should already be configured and will
 default to `admin@localhost`. If not, please go back and configure it now.
```
Comment (Hinton) [inline docs/getting-started/server/admin.md:12]: Do we want to update https://github.com/bitwarden/server/blob/master/dev/secrets.json.example as well?

File: docs/getting-started/server/admin.md [hunk 3]
```diff
@@ -1,19 +1,51 @@
...
+
+If you wish to change the membership for any role, you can edit your user secrets to specify the
+desired value.
+
+:::info
+
+Role-based access control is only enforced on cloud-hosted instances. There is no role-based access
+control on self-hosted deployments.
+
+:::
```
Comment (Hinton) [inline docs/getting-started/server/admin.md:47]: That's an odd decision. Which value do we read from on SH now?

File: docs/getting-started/server/admin.md
Comment (Hinton) [inline docs/getting-started/server/admin.md:1]: Do we want to rename this file to `portal`?

File: docs/getting-started/server/admin.md (hunk 1 above)
Comment (trmartin4) [reply docs/getting-started/server/admin.md:8]: Got it.  I've updated the `adminSettings:admins` as well to include all the emails.

File: docs/getting-started/server/portal.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+
+If you wish to change the membership for any role, you can
+[edit your user secrets](./user-secrets.md) to specify the desired value.
+
+:::info
+
+Role-based access control is only enforced on cloud-hosted instances. There is no role-based access
+control on self-hosted deployments.
+
+:::
```
Comment (Hinton) [inline docs/getting-started/server/portal.md:55]: We should move this higher up, or put the whole block under a `<bitwarden>` tag to hide this for community contributors.

Comment (trmartin4) [reply docs/getting-started/server/portal.md:55]: I've added the `<bitwarden>` tags around the entire authorization section.

The `secrets.json.example` is a bit misleading for community contributors, as we've defined roles there that they won't use.  I'm not sure if that is an issue though.

**Review by Hinton — APPROVED** [review]

**Pull request merged**